src/Features/Core/Portable/MetadataAsSource/MetadataAsSourceOptions.cs

```diff
@@ -12,13 +14,14 @@ namespace Microsoft.CodeAnalysis.MetadataAsSource
     /// <param name="AlwaysUseDefaultSymbolServers">Whether navigation should try to use the default Microsoft and
     /// Nuget symbol servers regardless of debugger settings</param>
     internal readonly record struct MetadataAsSourceOptions(
+        SimplifierOptions? SimplifierOptions = null,
         bool NavigateToDecompiledSources = true,
         bool AlwaysUseDefaultSymbolServers = true)
     {
         public static readonly MetadataAsSourceOptions Default = new();
 
         public MetadataAsSourceOptions()
-            : this(NavigateToDecompiledSources: true)
+            : this(SimplifierOptions: null)
```

Comment:
aside: i'm not really a fan of using structs here primarily because i find having to reason about no-param constructors (vs, all optional constructors) so painful.  Consider moving these to classes.